### ISSUE TITLE
F-DROID

### ISSUE THREAD

**Issue by fabianski7** [simple-login/Simple-Login-Android#1]
Now that the app has been released, I think it can be uploaded to the f-droid repository as well.

**Comment by ntnhon** [comment]
We are considering because fdroid doesn't allow an app that has tracker, in our case crashlytics & analytics. These tools help us understand user behaviours (anonymously of course) and most importantly crash reports.

**Comment by fabianski7** [comment]
In another issue, @nguyenkims said there would be no trackers in the app.

simple-login/app#117
> [...] There's no tracker in our app actually.

You could keep the trackers (since you said they collect anonymous data) in the version available on the google play and remove them from the version for f-droid.

**Comment by ntnhon** [comment]
@fabianski7: we will make the app compatible with F-Droid as you suggested.

I'm refactoring some parts of the project along with analytic removals. I will have you updated with the progress.

**Comment by ntnhon** [comment]
@fabianski7: I filed an inclusion request here: [https://gitlab.com/fdroid/rfp/-/issues/1344](https://gitlab.com/fdroid/rfp/-/issues/1344)

**Comment by marco3mt** [comment]
An alternative to Google Analytics and Crashlytics is the one used by the Official F-droid app itself:

https://github.com/ACRA/acra

https://inthecheesefactory.com/blog/how-to-install-and-use-acra-android/en

**Comment by ntnhon** [comment]
@ghost: this is because it scans the first 2 releases.

Thank you for the suggestion but we decided to use [Sentry](https://sentry.io) for crash reports because it doesn't require self-host and it supports both iOS and Android.

I configured the app to be flavored into 2 versions. The one for playstore with Sentry and the one for Fdroid without it. The app permissions are now reduced to just 1 (INTERNET). Facebook & Google related stuffs are completely removed so tracker count is 0.

The app is approved by Fdroid maintainer and will be published soon. I will keep you guys updated.
